tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/GitHub/GitHubLabelsTool.cs

```diff
@@ -97,12 +97,19 @@ public async Task<ServiceLabelResponse> CheckServiceLabel(string serviceLabel)
 
         private async Task<LabelHelper.ServiceLabelStatus> getServiceLabelInfo(string serviceLabel)
         {
-            logger.LogInformation("Checking service label: {serviceLabel}", serviceLabel);
-
             var csvContents = await githubService.GetContentsSingleAsync(Constants.AZURE_OWNER_PATH, Constants.AZURE_SDK_TOOLS_PATH, Constants.AZURE_COMMON_LABELS_PATH);
 
             var result = LabelHelper.CheckServiceLabel(csvContents.Content, serviceLabel);
+            if (result == LabelHelper.ServiceLabelStatus.Exists)
+            {
+                return result;
+            }
 
+            var pullRequests = await githubService.SearchPullRequestsByTitleAsync("Azure", "azure-sdk-tools", "Service Label");
```

Comment:
Consider whether search is needed, if it is needed maybe use something more narrowly defined like head branch.

https://github.com/Azure/azure-sdk-tools/issues/11778